cranelift/codegen/src/machinst/lower.rs

```diff
@@ -122,6 +125,29 @@ pub struct Lower<'func, I: VCodeInst> {
     next_vreg: u32,
 }
 
+/// Notion of "relocation distance". This gives an estimate of how far away a symbol will be from a
+/// reference.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum RelocDistance {
+    /// Target of relocation is "nearby". The threshold for this is fuzzy but should be interpreted
+    /// as approximately "within the compiled output of one module"; e.g., within AArch64's +/-
+    /// 128MB offset. If unsure, use `Far` instead.
```

Comment:
I think this comment ought to complete the documentation of `colocated` in the `ExtFuncData` data structure, because it's more likely CL users will look at `ExtFuncData` rather than `RelocDistance`.

Reply:
Done -- added an additional note to both `colocated` flags, in both `ExtFuncData` and `GlobalValueData::Symbol`.